proto/go_client/config.pb.go

```diff
@@ -470,6 +470,8 @@ type ComputeTemplate struct {
 	GpuAccelerator string `protobuf:"bytes,6,opt,name=gpu_accelerator,json=gpuAccelerator,proto3" json:"gpu_accelerator,omitempty"`
 	// Optional pod tolerations
 	Tolerations []*PodToleration `protobuf:"bytes,7,rep,name=tolerations,proto3" json:"tolerations,omitempty"`
+	// Optional. Number of efas
+	Efa uint32 `protobuf:"varint,8,opt,name=efa,proto3" json:"efa,omitempty"`
```

Comment:
Is it worth generalizing this to a list of custom accelerators that are added to container resources? How long do we think this list will grow over time?

Reply:
Sounds good, i generalized this as extended resources. PTAL